ts/Data/DataTable.ts

```diff
@@ -159,7 +159,10 @@ class DataTable extends DataTableCore implements DataEvent.Emitter<DataTable.Eve
 
     public constructor(options: DataTableOptions = {}) {
         super(options);
+
         this.modified = this;
+        // Remove the modified property to prevent the circular reference.
+        delete this.modified.modified;
```

Comment:
Even if we wanted to have it, it would be redundant because you call `super` first.